.changeset/cold-bugs-try.md

```diff
@@ -0,0 +1,8 @@
+---
+"@directus/app": patch
```

Comment:
Since it's a breaking change, I guess for `@directus/app`
```suggestion
"@directus/app": major
```

Reply:
Is it a breaking change for the app or api?

Reply:
Hmm, technically probably rather for the api 🤔

Reply:
Made it a `major` change for `@directus/api`. Let me know if you feel like that doesn't make sense.